subscription-manager.spec

```diff
@@ -361,7 +361,7 @@ Requires:  %{py_package_prefix}-python-dateutil
 Requires: %{py_package_prefix}-dateutil
 %endif
 Requires: %{py_package_prefix}-iniparse
-Requires: subscription-manager-rhsm-certificates = %{version}-%{release}
+Requires: subscription-manager-rhsm-certificates
```

Comment:
Will we require a specific version in the future?

Reply:
My idea is not to, because the server-side certificates are hardly tied to the version of sub-man itself. Of course we can always change in the future if needed, however ATM I don't see the reason for it.

Reply:
works for me!